opcua/common/subscription.py

```diff
@@ -52,10 +52,21 @@ def __str__(self):
         return "EventResult({})".format([str(k) + ":" + str(v) for k, v in self.__dict__.items()])
     __repr__ = __str__
 
+    def get_field_variants(self):
```

Comment:
can you give that method a more understanble name? since it is now exposed. get_event_properties_as_field_list() or whatever make sense